worlds/aquaria/Options.py

```diff
@@ -34,9 +34,30 @@ class TurtleRandomizer(Choice):
     default = 2
 
 
-class EarlyEnergyForm(DefaultOnToggle):
-    """ Force the Energy Form to be in a location early in the game """
-    display_name = "Early Energy Form"
+class EarlyBindSong(Choice):
+    """
+    Force the Bind song to be in a location early in the multiworld (or directly in your world if Early and Local is
+    selected).
+    """
+    display_name = "Early Bind song"
+    option_off = 0
+    option_early = 1
+    option_early_and_local = 2
+    default = 1
+
+
+class EarlyEnergyForm(Choice):
+    """
+    Force the Energy form to be in a location early in the multiworld (or directly in your world if Early and Local is
+    selected).
+    """
+    display_name = "Early Energy form"
+    option_off = 0
+    option_early = 1
+    option_early_and_local = 2
+    default = 1
```

Comment:
I forget if it's possible with yaml parsing overrides, but would be nice to add an 'on' alias so existing yamls can be used

Reply:
You'd want to cover `true` as well

Reply:
How do I do that without having 2 options names for the same option values?

Reply:
Here's an example in Witness https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/witness/options.py#L60-L66

Reply:
Ok. Thanks. Will do that.